### ISSUE TITLE
Improve English in the comments of the main header - [merged]

### ISSUE THREAD

**Issue by iskra-anl** [anlsys/excit#18]
In GitLab by @iskra-anl on Apr 10, 2020, 10:42

_Merges comment-const-tweaks -> master_

Also do some minor tweaking of const.

Some additional comments regarding the excit.h header file that may require additional work:

* "stutters"

Is that an established term?  If not, can we replace it with a less... loaded... one?

* "const excit_t"

Since excit_t is a pointer type, the const will apply to the pointer, not to the contents behind it.  I don't think there's really any way to solve it, short of typedef'ing a separate const_excit_t type, something like (not tested):

`typedef const struct excit_s *const_excit_t;`

* "next functionality"

The comment is hard to parse; maybe replace "next" with something less generic (just in the comment; it's fine as the field name)?  I chose "succession" but maybe "advancement" or "incrementation" would be better?  Maybe also use more descriptive terms in the comments for "nth" and "pos"?

**Comment by iskra-anl** [comment]
In GitLab by @iskra-anl on Apr 10, 2020, 10:46

The pipelines have failed, but that seems to be because of CI configuration issues?

**Comment by iskra-anl** [comment]
In GitLab by @Kerilk on Apr 10, 2020, 11:01

assigned to @freuk and unassigned @Kerilk

**Comment by iskra-anl** [comment]
In GitLab by @Kerilk on Apr 10, 2020, 11:02

assigned to @freuk and unassigned @Kerilk

**Comment by iskra-anl** [comment]
In GitLab by @Kerilk on Apr 10, 2020, 11:03

Thanks for the feedback Kamil, and all the corrections.
I will remove all the extra const for now and I'll think about bringing constification back.
For stutters I'll change it to something else.

I am no expert in CI, I'll tag @freuk, maybe he knows what went wrong.

**Comment by iskra-anl** [comment]
In GitLab by @Kerilk on Apr 10, 2020, 11:04

merged

**Comment by iskra-anl** [comment]
In GitLab by @Kerilk on Apr 10, 2020, 11:04

mentioned in commit 9f2c43e4fbe9b76c5754dc376ea08977b84036bd

**Comment by freuk** [comment]
Is your CI issue fixed or do I need to take a look? :)

**Comment by Kerilk** [comment]
Hey Val, this issue used to be fixed so I think you're fine :)
